src/Tribe/Views/V2/Assets.php

```diff
@@ -42,7 +42,7 @@ public function register() {
 			$plugin,
 			'tribe-events-views-v2-manager',
 			'views/manager.js',
-			[ 'jquery', 'tribe-common', 'tribe-query-string' ],
+			[ 'jquery', 'tribe-common', 'tribe-query-string', 'underscore' ],
```

Comment:
Does WP ship with underscore? If not, can we pull in modules as required from our code instead to not download the full underscore library?

Reply:
WordPress ship with Underscore.